### DISCUSSION TITLE
Welcome to Community Discussions!

### DISCUSSION THREAD

**Discussion by ajeetraina** [docker-community-leaders/community#17] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our Docker community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sangam14** [comment]
Now its exciting I'm using via github mobile 📱 app its like using slack or discord. 😊

**Comment by ajeetraina** [comment]
I am sure community is going to love this

**Comment by ajeetraina** [comment]
Few of high priority items includes:
- Revamping theme
- Focusing on Docker, Python and Go Workshop by the end of this month(technical and non-technical both.)

**Reply by sangam14** [reply]
@gauravgahlot @ajeetraina can we move docker intro part from go docker workshop track! Let's build TOC for each track! That's help lot to contribute! Feel free to reply in thread 👇

**Comment by ajeetraina** [comment]
That's right @sangam14 Let me add TOC

**Reply by gauravgahlot** [reply]
That's a good idea. I started moving common things out of Go/Java workshop. A ToC will definitely help.

**Reply by ajeetraina** [reply]
I have added "Before the workshop" structure layout. https://docker-community-leaders.github.io/alphasite/handbooks/docker101/
